pkg/kapp/resources/resources.go

```diff
@@ -96,6 +96,9 @@ func (c *ResourcesImpl) All(resTypes []ResourceType, opts AllOpts) ([]Resource,
 		opts.ListOpts = &metav1.ListOptions{}
 	}
 
+	// Populate FallbackAllowedNamespace with resource namespaces stored during deploy
+	c.opts.FallbackAllowedNamespaces = uniqAndValidNamespaces(append(c.opts.FallbackAllowedNamespaces, opts.ResourceNamespaces...))
```

Comment:
Does this not mean that `--dangerous-scope-to-fallback-allowed-namespaces` does not do anything unless only one ns is being used? Do we want it to handle multiple namespaces?

I do not recall when exactly it comes into play.

Reply:
I am not sure what you mean here 😓

Reply:
Food for thought:
When do we use whatever is happening in line 103. Is the dangerous flag only to be used when all resources belong to one namespace and we wanna scope to that?

Or do we wanna change things to accommodate multiple namespaces